main.py

```diff
@@ -42,10 +40,27 @@ def parse_date(datestr):
         )
 
 
+def send_command(command):
+    process = subprocess.Popen(
+        command,
+        shell=True,
+        stdout=subprocess.PIPE
+    )
+    logger.debug(
+        f'{command} was executed.'
+    )
+    output, error = process.communicate()
+
+    if error:
+        raise Exception(error)
+
+    return json.loads(output.decode('utf-8'))
+
+
 @dataclasses.dataclass
 class Author:
     login: str
-    name: str  = dataclasses.field(default=None)
+    name: str = dataclasses.field(default=None)
```

Comment:
I think `gh` doesn't return the `name` for the `author` object. Good to have this though in case that changes!